src/java/org/mqttkat/server/MqttServer.java

```diff
@@ -2,8 +2,11 @@
 
 import static java.nio.channels.SelectionKey.OP_ACCEPT;
 
-import java.nio.channels.*;
+import java.nio.channels.Selector;
+import java.nio.channels.ServerSocketChannel;
 import java.net.InetSocketAddress;
+import java.nio.channels.SelectionKey;
+import java.nio.channels.SocketChannel;
 
```

Comment:
This needs to be imported as well: import java.nio.channels.*;

for the exception: ClosedSelectorException